docusaurus.config.ts

```diff
@@ -0,0 +1,141 @@
+import { themes as prismThemes } from "prism-react-renderer";
+import type { Config } from "@docusaurus/types";
+import type * as Preset from "@docusaurus/preset-classic";
+
+const config: Config = {
+  title: "Mercurius",
+  tagline: "Mercurius is a GraphQL adapter for Fastify",
+  favicon: "favicon.ico",
+
+  // Set the production url of your site here
+  url: "https://your-docusaurus-site.example.com",
```

Comment:
Might be worth changing this

Reply:
I have updated the configuration, see PR here: https://github.com/mercurius-js/website/pull/1/commits/54d8a2dc7ab12c4165ec309fd51e6865fc432520